test/js/pageobjects/Welcome.js

```diff
@@ -7,6 +7,8 @@ class Welcome extends Base {
   get subtitle() { return this.$('.onfido-sdk-ui-Welcome-text')}
   get footer() { return this.$('.onfido-sdk-ui-Theme-footer')}
   get primaryBtn() { return this.$('.onfido-sdk-ui-Button-button')}
+  get openModalButton() { return this.$('#button')}
+  get closeButton() { return this.$('.onfido-sdk-ui-Modal-closeButton')}
```

Comment:
Minor: recommend this is renamed to `closeModalButton` for consistency & to be clear what it is/does